website/docs/guides/test-your-typegraph/index.mdx

```diff
@@ -0,0 +1,113 @@
+import TabItem from "@theme/TabItem";
+import SDKTabs from "@site/src/components/SDKTabs";
+
+import PyPmTabs from "@site/src/components/PythonPackageManagerTabs";
+import TsPmTabs from "@site/src/components/TsPackageManagerTabs";
+
+# Test typegraphs
+
+[Typegraph](/docs/reference/typegraph) is the frontend of the _Metatype_ ecosystem where the app is authored and defined. Typegraph is basically a function that includes the definition of the exposed APIs, the embedded runtimes, database table definitions, functions and other building blocks of the application.
+
+Typegraphs can be tested easily with the common test suites in the language of your choosing.
+
+The following next lines describes how you can test your typegraph.
+For _Typescript_ SDK, we will be using [vitest](https://vitest.dev/) and [pytest](https://docs.pytest.org/en/stable/) for the _Python_ SDK in this example.
+
+<SDKTabs>
+
+    <TabItem value="typescript">
+    example typegraph
+    </TabItem>
+
+    <TabItem value="python">
+    
+First, you need to add _pytest_ as a dev dependency in your project/application. 
+:::info
+You can follow [this](/docs/tutorials/quick-start) link on how to bootstrap a metatype application.
+:::
+<PyPmTabs>
+    <TabItem value="pip">
+    ```bash
+    pip3 install pytest
+    ```
+    </TabItem>
+    <TabItem value="poetry">
+    ```bash
+    poetry add --dev pytest
```

Comment:
I believe this should be --group dev now

Reply:
looks like it is indeed deprecated.